v2/solver.py

```diff
@@ -62,7 +62,11 @@ def importUI(self, ui):
             data.append([item, qty])
         for row in data:
             try:
-                name = row[0].lower().replace(' ','_')
+                name = row[0].lower().replace(' ','_').replace('slimefun:','slimefun4:')
```

Comment:
This is not the proper place to put this. I have already implemented a system of defaults in the appropriate place. Please drop this change.

Reply:
fixed in 'dropped slimefun to slimefun4 replacement' commit